assume/common/forecasts.py

```diff
@@ -9,6 +9,9 @@
 
 from assume.common.fast_pandas import FastIndex, FastSeries
 
+import warnings
+
+warnings.simplefilter(action='ignore', category=pd.errors.PerformanceWarning)
```

Comment:
This is a workaround to silence warnings when adding a series to an existing DataFrame multiple times in a loop.
It is better to collect the series to be added and join them in one action to the dataframe